packages/otel/package.json

```diff
@@ -8,29 +8,25 @@
   },
   "dependencies": {
     "@opentelemetry/api": "../../node_modules/@opentelemetry/api",
-    "@opentelemetry/api-logs": "../../node_modules/@opentelemetry/api-logs",
-    "@opentelemetry/core": "../../node_modules/@opentelemetry/core",
-    "@opentelemetry/exporter-logs-otlp-http": "../../node_modules/@opentelemetry/exporter-logs-otlp-http",
-    "@opentelemetry/exporter-trace-otlp-http": "../../node_modules/@opentelemetry/exporter-trace-otlp-http",
-    "@opentelemetry/instrumentation-http": "../../node_modules/@opentelemetry/instrumentation-http",
-    "@opentelemetry/instrumentation-winston": "../../node_modules/@opentelemetry/instrumentation-winston",
     "@opentelemetry/resources": "../../node_modules/@opentelemetry/resources",
-    "@opentelemetry/sdk-logs": "../../node_modules/@opentelemetry/sdk-logs",
-    "@opentelemetry/sdk-node": "../../node_modules/@opentelemetry/sdk-node",
-    "@opentelemetry/sdk-trace-base": "../../node_modules/@opentelemetry/sdk-trace-base",
-    "@opentelemetry/sdk-trace-node": "../../node_modules/@opentelemetry/sdk-trace-node",
     "@opentelemetry/semantic-conventions": "../../node_modules/@opentelemetry/semantic-conventions",
-    "eslint": "../../node_modules/eslint",
-    "typescript": "5.5.4",
-    "urql": "4.0.4"
+    "@opentelemetry/sdk-trace-node": "../../node_modules/@opentelemetry/sdk-trace-node",
+    "@opentelemetry/exporter-trace-otlp-http": "../../node_modules/@opentelemetry/exporter-trace-otlp-http",
+    "urql": "4.0.4",
+    "@saleor/app-sdk": "link:../../node_modules/@saleor/app-sdk"
   },
   "devDependencies": {
-    "@saleor/app-sdk": "link:../../node_modules/@saleor/app-sdk",
+    "typescript": "5.5.4",
+    "eslint": "../../node_modules/eslint",
     "eslint-config-saleor": "workspace:*"
   },
-  "main": "index.ts",
-  "peerDependencies": {
-    "@saleor/app-sdk": "link:../../node_modules/@saleor/app-sdk"
-  },
-  "type": "module"
+  "type": "module",
+  "exports": {
+    "./batch-span-processor-factory": "./src/batch-span-processor-factory.ts",
+    "./http-instrumentation-factory": "./src/http-instrumentation-factory.ts",
+    "./resource-factory": "./src/resource-factory.ts",
+    "./observability-attributes": "./src/observability-attributes.ts",
+    "./otel-urql-exchange-factory": "./src/otel-urql-exchange-factory.ts",
+    "./wrap-with-span-attributes": "./src/wrap-with-span-attributes.ts"
+  }
```

Comment:
I think we don't even need this? I mean it doesn hurt, but it's local dependency so it makes a synthetic alias on something so simple?

If importing from `src/` is strange, maybe we can flatten it or rename to `lib` or something

Reply:
Removed in [5437c23](https://github.com/saleor/apps/pull/1740/commits/5437c23a9c7a7906193afb7b9ba3b41443306518)